test/runtests.jl

```diff
@@ -27,7 +27,7 @@ end
 
 facts("oauth_sign_hmac_sha1") do
     context("provides a consistent string") do
-        expected = "WqKCG5iyhFiES3fWYVdWJWwinaY="
+        expected = "dR8PcWvTl2FyvVM417iTe/p1XV8="
```

Comment:
Just so I'm clear, was this original test incorrect?

Reply:
I believe so, yes. I checked several online hmac generators and they all have `WqKCG5iyhFiES3fWYVdWJWwinaY=` as the base64 result.

Reply:
@quinnj I'm almost certain this is the cause of #21. When the switch from Nettle to Mbedtls was made, I let you switch this test, but Twitter.jl is broken and #21 doesn't work any more.

Is it possible that "sha1" and "MD_SHA1" aren't the same algorithm?